lib/queries/otu/autocomplete.rb

```diff
@@ -26,6 +26,10 @@ class Autocomplete < Query::Autocomplete
       #   if 'true' then only #name = query_string results are returned (no fuzzy matching)
       attr_accessor :exact
 
+      # @return Integer, nil
+      # Only return otus tagged with this tag id
+      attr_accessor :project_scope_tag_id
```

Comment:
I'm worried about the scope in various other places, there are various other filter results that need this scoping that are used.  Before we get too far I'd like to enumerate where else we'd need this tag param to ensure consistency across the site.  Remember too that for basic OTUs this works fine, but as as soon as we add all the filters/search that others want to use this will be quite difficult to maintain without a lot of thought.